--- kwine-0.1/src/wcmd.desktop	2006-05-27 08:39:10.000000000 -0230
+++ kwine-0.1/src/wcmd.desktop.new	2007-03-15 01:19:35.000000000 -0230
@@ -6,4 +6,4 @@
 [Desktop Action openCMDHere]
 Name=Open CMD.EXE Here
 Icon=konsole
-Exec=konsole -e wine --workdir %F wcmd.exe
+Exec=konsole --workdir %f -e wine cmd.exe
--- kwine-0.1/src/kwine.cpp	2006-05-27 08:39:10.000000000 -0230
+++ kwine-0.1/src/kwine.cpp.new	2007-03-15 01:56:06.000000000 -0230
@@ -32,7 +32,7 @@
 /*
 #define WINECMD "/opt/cxoffice/bin/wine "
 */
-#define WINECMD "wine "
+#define WINECMD "wine"
 
 static const char description[] =
 I18N_NOOP("A KDE KPart Application");
@@ -83,8 +83,8 @@
 		KRun::runCommand( wine, name, icon );
 		QTimer::singleShot(2000, this, SLOT(deref()));
 	} else if ("application/x-bat" == mt->name()) {
-//		QString wine = "konsole --noclose -e " WINECMD " --workdir '" + workingDir.pathOrURL() + "' wcmd.exe /c '" + cmd.pathOrURL()+"'";
-		QString wine = "konsole --noclose  --workdir '" + workingDir.pathOrURL() + " -e " WINECMD " wcmd.exe /c '" + cmd.pathOrURL()+"'";
+//		QString wine = "konsole --noclose -e " WINECMD " --workdir '" + workingDir.pathOrURL() + "' cmd.exe /c '" + cmd.pathOrURL()+"'";
+		QString wine = "konsole --noclose  --workdir '" + workingDir.pathOrURL() + "' -e " WINECMD " cmd.exe /c '" + cmd.pathOrURL()+"'";
 		QString icon = "wine";
 		QString name = cmd.fileName();
 		
@@ -118,7 +118,7 @@
 			return exec();
 		} else if (( "application/x-bat" == mt->name()) && !info.isNetworkPath) {
 		
-		        QString wine = "konsole --noclose --workdir '" + workingDir.pathOrURL() +" -e " WINECMD " wcmd.exe /c '" + cmd.pathOrURL()+"'";
+		        QString wine = "konsole --noclose --workdir '" + workingDir.pathOrURL() +"' -e " WINECMD " cmd.exe /c '" + cmd.pathOrURL()+"'";
 			QString icon = "wine";
 			QString name = newurl.fileName();
 		        for (i=1;i< args_count;i++) wine += " \"" + QString(args->arg(i)) +"\"" ; 
