--- src/vte.c   2007-03-08 09:47:15.000000000 -0700
+++ src/vte.c   2007-03-21 10:02:19.000000000 -0600
@@ -1967,6 +1967,8 @@
 
 	screen = terminal->pvt->screen;
 
+	_vte_terminal_ensure_row (terminal);
+
 	/* The total number of lines.  Add one to the cursor offset
 	* because it's zero-based. */
 	rows = MAX(_vte_ring_next(terminal->pvt->screen->row_data),
@@ -9773,12 +9775,10 @@
 			row * height + VTE_PAD_WIDTH,
 			(col_stop - col) * width,
 			(row_stop - row) * height);
-	if (!GTK_WIDGET_DOUBLE_BUFFERED (terminal) ||
-			_vte_draw_has_background_image (terminal->pvt->draw)) {
-		_vte_draw_clear (terminal->pvt->draw,
-				area->x, area->y,
-				area->width, area->height);
-	}
+
+	_vte_draw_clear (terminal->pvt->draw,
+		area->x, area->y,
+		area->width, area->height);
 
 	/* Now we're ready to draw the text.  Iterate over the rows we
 	 * need to draw. */
