diff -ur mod_perl-2.0.9-orig/ModPerl-Registry/t/conf/extra.conf.in mod_perl-2.0.9/ModPerl-Registry/t/conf/extra.conf.in
--- mod_perl-2.0.9-orig/ModPerl-Registry/t/conf/extra.conf.in	2015-08-06 22:06:49.738009495 +0000
+++ mod_perl-2.0.9/ModPerl-Registry/t/conf/extra.conf.in	2015-08-06 22:10:35.377297791 +0000
@@ -219,8 +219,8 @@
 
     RewriteEngine On
     <IfVersion < 2.4.0>
-        RewriteLogLevel 9
-        RewriteLog @ServerRoot@/logs/rewrite_log
+        LogLevel rewrite:trace8
+        ErrorLog @ServerRoot@/logs/rewrite_log
     </IfVersion>
     RewriteRule /rewritetest /rewrite_env/env_val.pl?REWRITE_TEST [E=REWRITE_TEST:GOTCHA,PT,NS,L]
 
