diff -ur httpd24-orig/t/response/TestModules/proxy.pm httpd24/t/response/TestModules/proxy.pm
--- httpd24-orig/t/response/TestModules/proxy.pm	2013-06-21 13:11:41.421816434 +0000
+++ httpd24/t/response/TestModules/proxy.pm	2013-06-21 13:14:00.011312328 +0000
@@ -47,9 +47,7 @@
     <IfModule mod_proxy.c>
         <Proxy http://@servername@:@port@/*>
             <IfModule @ACCESS_MODULE@>
-                Order Deny,Allow
-                Deny from all
-                Allow from @servername@
+                Require host @servername@
             </IfModule>
         </Proxy>
 
