diff -urN phpldapadmin-1.2.0.5/lib/emuhash_functions.php phpldapadmin-1.2.0.5-new/lib/emuhash_functions.php
--- phpldapadmin-1.2.0.5/lib/emuhash_functions.php	2010-01-30 04:21:20.000000000 -0100
+++ phpldapadmin-1.2.0.5-new/lib/emuhash_functions.php	2010-11-11 11:52:10.802038291 -0100
@@ -58,8 +58,8 @@
 		function openssl_hash($openssl_hash_id,$password_clear) {
 			global $emuhash_emu;
 
-			$current_magic_quotes = get_magic_quotes_runtime();
-			set_magic_quotes_runtime(0);
+			$current_magic_quotes = ini_get('magic_quotes_runtime');
+			ini_set('magic_quotes_runtime', 0);
 			$tmpfile = tempnam($emuhash_emu['tmpdir'],'emuhash');
 			$pwhandle = fopen($tmpfile,'w');
 
@@ -73,7 +73,7 @@
 			$pass = fread($prog,1024);
 			pclose($prog);
 			unlink($tmpfile);
-			set_magic_quotes_runtime($current_magic_quotes);
+			ini_set('magic_quotes_runtime', $current_magic_quotes);
 
 			return $pass;
 		}
