--- setup.py.old	2014-04-11 14:37:05.557343225 -0400
+++ setup.py	2014-04-11 14:37:30.560989056 -0400
@@ -248,7 +248,7 @@
                                      'utils', 'resource-path')
         with io.open(resource_file, 'w', encoding='utf-8',
                      errors='strict') as fp:
-            path = os.path.abspath(os.path.join(self.install_data, 'share'))
+            path = '/usr/share'
             if sys.version_info[0] < 3:
                 path = unicode(path)
             fp.write(path)
